lib/helpers/hasEmptyProperty.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Checks if there is any properties of @obj that is a empty object
+ * @function
+ * @param {object} obj - the object to check
+ */
+function hasEmptyProperty(obj) {
+  return Object.values(obj).every(
```

Comment:
`Object.values` does not exist in Node v6
This should result in a major/braking change version update so as to not break those still using Node v6

Reply:
Could change it too:

```suggestion
  return Object.keys(obj).map(key => obj[key]).every(
```

and consider it a bug fix. This shouldn't warrant a major version bump.

Or transpile the entire lib.

Reply:
No transpilations please. I had to think about that, but didn't. Most certainly it's not a major version bump.

Reply:
@bvanderlaan https://github.com/Surnet/swagger-jsdoc/pull/145